base.css

```diff
@@ -30,3 +30,8 @@ body {
 .vscode-body code > div {
     background: none
 }
+
+vscode-body table th,
+.vscode-body table td {
+    border: 1px solid #ddd !important;
```

Comment:
Instead of adding `!important`, can you write a more specific css rule to override the existing one?

Reply:
Suppose we have markdown code:

```md
| A | B | C |
|--|--|--|
| 1 | 2 | 3 |
```

vsocde (markdown-it) renders:

```html
<article class="vscode-body">
    <table>
        <thead>
            <tr>
                <th>A</th>
                <th>B</th>
                <th>C</th>
            </tr>
        </thead>
        <tbody>
            <tr>
                <td>1</td>
                <td>2</td>
                <td>3</td>
            </tr>
        </tbody>
    </table>
</article>
```

It's so clean that we have no other attributes to assign the rules. So I cannot find another way to fix, except `!important`. If another way existed, it must be beyond my capabilities.